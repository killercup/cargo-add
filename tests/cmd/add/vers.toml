bin.name = "cargo-add"
args = ["add", "my-package:>=0.1.1"]
status = "success"
stdout = ""
stderr = """
      Adding my-package >=0.1.1 to dependencies.
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
