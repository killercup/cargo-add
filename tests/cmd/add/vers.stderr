      Adding my-package v>=0.1.1 to dependencies. 
